VERSION

```diff
@@ -1 +1 @@
-1!2.1.3
+1!2.1.4
```

Comment:
Taking into account this is technically a breaking change, could you please bump the major version instead?

```suggestion
1!3.0.0
```